Utilities/src/main/java/edu/stanford/hivdb/utilities/FastaUtils.java

```diff
@@ -60,34 +60,28 @@ private FastaUtils() {}
 	 * @return InputStream
 	 */
 	private static InputStream cleanup(final InputStream stream) {
-		final BufferedReader reader =
-			new BufferedReader(new InputStreamReader(stream));
+		final BufferedReader reader = new BufferedReader(new InputStreamReader(stream));
 		List<String> result = new ArrayList<>();
 		boolean isPrevIdentLine = false;
-			
+
 		for (String line : reader.lines().toArray(size -> new String[size])) {
-			if (line.startsWith("#")) {
-				continue;
-			}
-			if (line.startsWith(">")) {
-				if (isPrevIdentLine) {
-					result.remove(result.size() - 1);
-				}
-				isPrevIdentLine = true;
-			}
-			else {
-				isPrevIdentLine = false;
-			}
+			if (line.startsWith("#")) continue;
+			if (line.startsWith(">") && isPrevIdentLine) {
```

Comment:
I kind of prefer the old implementation since we only call `startsWith(">")` once. However it's fine (so don't change them back).